Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -365,10 +374,10 @@ Function Install-LIS-Scenario-6 ($PreviousTestResult, $LISTarballUrlOld, $LISTar
 # Expected result : Verify that LIS has upgraded to Current LIS successfully after kernel upgrade
 Function Install-LIS-Scenario-7 ($PreviousTestResult, $LISTarballUrlOld, $LISTarballUrlCurrent) {
     if ($PreviousTestResult -eq "PASS") {
-        $is_oracle = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort -username "root" -password $password -command "cat /etc/os-release | grep -i oracle" -ignoreLinuxExitCode:$true
-        if ($is_oracle) {
+        $detectedDistro = Detect-LinuxDistro -VIP $allVMData.PublicIP -SSHport $allVMData.SSHPort -testVMUser $user -testVMPassword $password
```

Comment:
you don't need this, detectDistro is a global var and you can use it directly from there.

Reply:
Removed calling again Detect-LinuxDistro function and directly used detectDistro